scss/sections/_header.scss

```diff
@@ -1,282 +1,241 @@
+
+    &.show {
+        transition: opacity .2s ease, width 0s;
+        opacity: 1;
+        visibility: visible;
+    }
+
+    &:before {
+        position: fixed;
+        right: 12.5%;
+        top: calc-em(35);
+        font-size: calc-em(20);
+    }
+}
+
+.mobile-menu {
+    visibility: hidden;
+    transform: translateX(-100%);
+    top: 0;
+    left: 0;
+    z-index: $zindex-fixed;
+    overflow-x: hidden;
+    transition: visibility .2s ease-in-out, transform .2s ease-in-out;
+    padding-left: calc-em(15);
+    padding-right: calc-em(15);
+
+    &.show {
+        visibility: visible;
+        transform: translateX(0);
+    }
+
+    .card {
+        border: none;
+    }
+
+    .card-header {
+        border-bottom: none;
+        &:before {
+            font-size: calc-em(13);
+            transform: translateY(-50%) rotate(180deg);
+        }
+    }
+
+    .card-body--border {
+        border-top: calc-em(5) solid $gray-500;
+        &-hair {
+            border-color: $secondary-light;
+        }
+
+        &-tan {
+            border-color: $sh-purple-light;
+        }
+
+        &-body {
+            border-color: $bali-bod-blue-light;
+        }
+    }
+
+    .color-hair {
+        color: $secondary;
+    }
+
+    .color-tan {
+        color: $sh-purple;
+    }
+
+    .color-body {
+        color: $bali-bod-blue;
+    }
+
+    .btn {
+        font-size: calc-em(20);
+        padding-top: calc-em(5.5);
+        padding-bottom: calc-em(5.5);
+        border-radius: calc-em(6);
+    }
+
+    h5 {
+        font-size: calc-em(26);
+    }
+
+    a {
+        color: $body-color;
+        &:focus,
+        &:hover {
+            text-decoration: none;
+        }
+    }
+
+    img {
+        border-radius: calc-em(6);
+    }
+}
+
+.nav-item {
+    padding-left: calc-em(8);
+    padding-right: calc-em(8);
+    &:first-child {
+        padding-left: 0;
+    }
+
+    &:last-child {
+        padding-right: 0;
+    }
+}
+
+.navbar-nav .nav-link {
+    font-weight: 700;
+    text-transform: uppercase;
+    @include media-breakpoint-up(xl) {
+        font-size: calc-em(20);
+    }
+
+    &.dropdown-toggle {
+        padding: calc-em(6.5, 20) calc-em(17, 20);
+        border-radius: calc-em(6);
+    }
+
+    &:hover {
+        text-decoration: none;
+    }
+}
+
+.navbar-toggler:focus {
+    outline: none;
+}
+
+.navbar-toggler-icon {
+    width: calc-em(25, 20);
+    height: 2px;
+    background-color: $body-color;
+    position: relative;
+    &::before,
+    &::after {
+        content: '';
+        height: 2px;
+        background: $body-color;
+        position: absolute;
+        left: 0;
+    }
+
+    &::before {
+        width: calc-em(21, 20);
+        top: -(calc-em(8, 20));
+    }
+
+    &::after {
+        width: calc-em(19, 20);
+        top: calc-em(8, 20);
+    }
+}
+
+@include media-breakpoint-up(lg) {
+    .dropdown:hover .dropdown-menu--mega-menu {
+        opacity: 1;
+        visibility: visible;
```

Comment:
we don't need visibility values

Reply:
@foria I tried to remove this visibility for desktop megamenu but there is an issue with hover